node-runner-cli/api/CoreApiHelper.py

```diff
@@ -159,6 +159,7 @@ def construction_build(self, actions, print_response=False, ask_user=False):
                 network_configuration: NetworkConfigurationResponse = self.network_configuration()
                 key_list: KeyListResponse = self.key_list()
                 operation_groups = ValidatorConfig.build_operations(actions, key_list, ask_user=ask_user)
+                if len(operation_groups) == 0: return
```

Comment:
I think it's more idiomatic, and a little clearer, to put blocks like this on their own lines.